.github/workflows/patch-release-build.yml

```diff
@@ -110,6 +110,9 @@ jobs:
 
       - name: Set up Docker Buildx
         uses: docker/setup-buildx-action@v3
+        with:
+          driver-opts: image=moby/buildkit:v0.15.1
+        if: ${{ matrix.os == 'ubuntu-latest' }}
```

Comment:
IMPORTANT:
Do not include this `if` condition. It is unnecessary and will cause the Docker build to be skipped since `matrix.os` is not defined here.

See issue and fix that this `if` statement cause here:
https://github.com/yiyuan-he/aws-otel-java-instrumentation/commit/0fdbf83bfb02b37fbc1d60e650af006998f20e29